src/flashbar/__tests__/flashbar.test.tsx

```diff
@@ -450,155 +441,3 @@ describe('Flashbar component', () => {
     testFlashDismissal({ stackItems: false });
   });
 });
-
-describe('Analytics', () => {
```

Comment:
This is a separate top-level describe block, which  naturally suggests it should be a separate test file in the first place